bigtable/beam/change-streams/src/main/java/ChangeStreamsHelloWorld.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START bigtable_beam_helloworld_read]
+import com.google.cloud.Timestamp;
+import com.google.cloud.bigtable.data.v2.models.ChangeStreamMutation;
+import com.google.cloud.bigtable.data.v2.models.ChangeStreamMutation.MutationType;
+import com.google.cloud.bigtable.data.v2.models.DeleteCells;
+import com.google.cloud.bigtable.data.v2.models.DeleteFamily;
+import com.google.cloud.bigtable.data.v2.models.Entry;
+import com.google.cloud.bigtable.data.v2.models.SetCell;
+import com.google.protobuf.ByteString;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.FlatMapElements;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.TypeDescriptors;
+
+public class ChangeStreamsHelloWorld {
+  public static void main(String[] args) {
+    BigtableOptions options =
+        PipelineOptionsFactory.fromArgs(args).withValidation().as(BigtableOptions.class);
+    Pipeline p = Pipeline.create(options);
+
+    final Timestamp startTime = Timestamp.now();
+    final Timestamp endTime =
+        Timestamp.ofTimeSecondsAndNanos(
+            startTime.getSeconds() + 5 * 60, // Add 10 minutes
+            startTime.getNanos());
+    p.apply(
+            "Read Change Stream",
+            BigtableIO.readChangeStream()
+                .withProjectId(options.getBigtableProjectId())
+                .withInstanceId(options.getBigtableInstanceId())
+                .withTableId(options.getBigtableTableId())
+                .withAppProfileId("default")
```

Comment:
nit: maybe we should parameterize this in case it doesn't have the correct configuration for the instance (single-cluster routing and single-row transactions)

Reply:
I was going to remove it once the bug with it being required was fixed, but I can add it as an option since it is probably useful to have documented